stubs/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "stubs"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+solana-program = "1.14.10"
```

Comment:
generic `stubs` should not depend on `solana-program`. Probably means this needs to be refactored.

Reply:
the file `solana-stubs.rs` tries to create `stubs` for 'Pubkey' and 'AccountInfo'. We should talk about this?

Reply:
the stubs are external functions that are called by `nondet` implementation. So perhaps they should be packaged with `nondet` rather than in a separate crate